docs/reference/vendor-api/index.md

```diff
@@ -14,4 +14,6 @@ url        = "/docs/reference/vendor-api"
 parent     = "/reference"
 +++
 
-This section of the docs explains how to use the Vendor API.
+The Vendor API is available for you to integrate with your business workflows, from automating your continuous integration pipeline to integrating your account creation scripts with Replicated license management.
```

Comment:
One variation:

The Vendor API is available for integrating Replicated into your business workflows for tasks such as automating your continuous integration pipeline or integrating Replicated license management into your account creation scripts.